### ISSUE TITLE
NCDFReader has no attribute skip

### ISSUE THREAD

**Issue by casawa** [MDAnalysis/mdanalysis#539]
I'm trying to use the density analysis on an AMBER Trajectory.

I run the following:
u = Universe(PDB, NCTrajectory)
D = density_from_Universe(u, delta=1.0, atomselection="name OH2")

Then, it histograms all the frames, and at the last frame, it says:
AttributeError: 'NCDFReader' object has no attribute 'skip'

**Comment by richardjgowers** [comment]
Hey

That's a bug we've recently fixed (see #489), as a workaround you can do:

```
u.trajectory.skip = 1
```

After you load the universe

**Comment by orbeckst** [comment]
Or you can [install the development version](https://github.com/MDAnalysis/mdanalysis/wiki/DevelopmentBranch).

Let us know if either of these methods work so that we can close this issue.

**Comment by casawa** [comment]
Thanks! I had to set a few more attributes as well, so I do something like this now:

u = Universe(PDB, NC file)
u.trajectory.skip = 1
u.trajectory.skip_timestep = 1
u.trajectory.delta = 1
D = density_from_Universe(u, delta=1.0, atomselection="name OH2")

Then, while exporting to a DX file, I get the following error: NotImplementedError: Only regularly spaced grids allowed. 

I set delta to be 1.0, so I was confused why this wouldn't be regularly spaced?
